landing-zone-submodule/management-vpc/variables.tf

```diff
@@ -0,0 +1,303 @@
+variable "region" {
+  description = "The region to which to deploy the VPC"
+  type        = string
+  default     = "au-syd"
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
+  type        = string
+  default     = "management"
+}
+
+variable "resource_group_id" {
+  description = "The resource group ID where the VPC to be created"
+  type        = string
+}
+
+variable "tags" {
+  description = "List of tags to apply to resources created by this module."
+  type        = list(string)
+  default     = []
+}
+
+
+#############################################################################
+# VPC variables
+#############################################################################
+
+variable "network_cidr" {
+  description = "Network CIDR for the VPC. This is used to manage network ACL rules for cluster provisioning."
+  type        = string
+  default     = "10.0.0.0/8"
+}
+
+variable "classic_access" {
+  description = "Optionally allow VPC to access classic infrastructure network"
+  type        = bool
+  default     = null
+}
+
+variable "use_manual_address_prefixes" {
+  description = "Optionally assign prefixes to VPC manually. By default this is false, and prefixes will be created along with subnets"
+  type        = bool
+  default     = true
+}
+
+variable "default_network_acl_name" {
+  description = "Override default ACL name"
+  type        = string
+  default     = null
+}
+
+variable "default_security_group_name" {
+  description = "Override default VPC security group name"
+  type        = string
+  default     = null
+}
+
+variable "default_routing_table_name" {
+  description = "Override default VPC routing table name"
+  type        = string
+  default     = null
+}
+
+variable "default_security_group_rules" {
+  description = "Override default security group rules"
+  type = list(
+    object({
+      name      = string
+      direction = string
+      remote    = string
+      tcp = optional(
+        object({
+          port_max = optional(number)
+          port_min = optional(number)
+        })
+      )
+      udp = optional(
+        object({
+          port_max = optional(number)
+          port_min = optional(number)
+        })
+      )
+      icmp = optional(
+        object({
+          type = optional(number)
+          code = optional(number)
+        })
+      )
+    })
+  )
+
+  default = []
+}
+
+variable "address_prefixes" {
+  description = "Use `address_prefixes` only if `use_manual_address_prefixes` is true otherwise prefixes will not be created. Use only if you need to manage prefixes manually."
+  type = object({
+    zone-1 = optional(list(string))
+    zone-2 = optional(list(string))
+    zone-3 = optional(list(string))
+  })
+
+  default = null
```

Comment:
I see some address prefixes in the management vpc created from landing-zone, so this seems missing here.

![image](https://user-images.githubusercontent.com/16447795/228956495-01b10b47-8f0d-4137-8f13-6e07d737e426.png)

Reply:
I tested using both no-compute example and the landing-zone example in the vpc. It created the same number of address prefixes with the same range.
The below vpc was created using the no-compute example in slz.
<img width="1324" alt="Screenshot 2023-04-01 at 12 48 50 PM" src="https://user-images.githubusercontent.com/122446118/229271843-5e7a5e58-de7f-4eb3-a5c5-bf2980455151.png">


and the below vpc is created using the landing zone example in the vpc module.
<img width="1258" alt="Screenshot 2023-04-01 at 12 48 59 PM" src="https://user-images.githubusercontent.com/122446118/229271866-37c531a2-b068-4da6-bd6c-2400f6ef990a.png">